test/server/models/Collective.test.js

```diff
@@ -428,7 +428,10 @@ describe('server/models/Collective', () => {
       website: 'https://opencollective.com',
     });
     // Make sure clearbit image is fetched (done automatically and async in normal conditions)
+    const originalValue = process.env.OC_ENV;
+    process.env.OC_ENV = 'ci-exception';
     await collective.findImage();
+    process.env.OC_ENV = originalValue;
```

Comment:
Changing the env dynamically is a bit dangerous: if `findImage` throws an exception the env will never be restored and other tests in the chain will break because of that. Plus, we don't know what kind of side-effect it can introduces if part of the targetted code starts to rely on the env too.

I would prefer having a `force` parameter (boolean) on `findImage` / `findImageForUser`. 

Note that though we don't respect it in the actual codebase, tests frameworks usually discourage checking the environment in the code like we do here. The best practice is usually to mock the function (or the [call](https://github.com/nock/nock) to the HTTP server) - globally if necessary.

Reply:
@Betree : Yes, I agree. Thanks for the information. I've corrected it by adding a force parameter. 👍🏽